docs/intro/quickstart/modeling.rst

```diff
@@ -0,0 +1,101 @@
+.. _ref_quickstart_modeling:
+
+=================
+Modeling the data
+=================
+
+.. edb:split-section::
+
+  The flashcards application has a simple data model, but it's interesting enough to get a taste of many of the features of the |Gel| schema language. You have a ``Card`` type that describes a single flashcard, which for now contains two required string properties: ``front`` and ``back``. Each ``Card`` belongs to a ``Deck``, and there is an explicit ordering to the cards in a given deck.
+
+  Looking at the mock data, you can see this structure in the JSON.
+
+  .. code-block:: typescript
+
+    interface Card {
```

Comment:
in the tutorial app we define `RawJSONCard` and `RawJSONDeck` in both `lib/models` and in the `app/model`, is that needed?

Reply:
Yeah, we don't need both, PRs welcome, but that's more of an implementation detail than something related to the quickstart text itself. We don't actually mention those model modules directly.